src/DynamoCoreWpf/Views/Core/NodeView.xaml

```diff
@@ -45,42 +45,37 @@
                           Header="{x:Static p:Resources.ContextMenuDelete}"
```

Comment:
Since `ArgumentLacing`, `IsVisible` and `IsUpstreamVisible` properties are read-only, we need to make the binding expression `OneWay`, otherwise an exception will be thrown by WPF data-binding mechanism.